drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/models/ScenarioGridModel.java

```diff
@@ -1029,10 +1033,13 @@ protected void checkValidAndUniquePropertyHeaderTitle(String propertyHeaderCellV
      * Verify if the given value is not already used as property header name <b>inside the same instance</b>
      * @param propertyHeaderCellValue
      * @param columnIndex
-     * @throws Exception if the given <b>propertyHeaderCellValue</b> has already been used
+     * @throws IllegalArgumentException if the given <b>propertyHeaderCellValue</b> has already been used
      * inside the <b>instance</b> of the given column
      */
-    protected void checkUniquePropertyHeaderTitle(String propertyHeaderCellValue, int columnIndex) throws Exception {
+    protected void checkUniquePropertyHeaderTitle(String propertyHeaderCellValue, int columnIndex) {
+        if (propertyHeaderCellValue.isEmpty()) {
```

Comment:
@yesamer
Is there any check for null on propertyHeaderCellValue somewhere? Could we add one here - even if currently seems unneeded?

Reply:
@gitgabrio done!